modules/op-results-search-records.adoc

```diff
@@ -0,0 +1,73 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="results-search-records_{context}"]
+= Searching for records
+
+[role="_abstract"]
+You can search for records using Common Expression Language (CEL) queries. As each record contains full YAML information for a pipeline run or task run, you can find records by many different criteria.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility and prepared its environment to quety {tekton-results}.
+
+.Procedure
+
+* To search for records using a CEL query, run the following command:
++
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} --filter=<CEL query> <namespace-name>/result/-
```

Comment:
@mramendi IMHO we could update this command to use double quotes like this `--filter="<CEL query>"`.